app/forms/schools/add_participants/base_wizard.rb

```diff
@@ -160,22 +160,36 @@ def email_in_use?
 
         return false if adding_ect_as_mentor?
 
+        return false if email_owner_participant_profile_withdrawn_from_current_school?
+        return false if email_owner_induction_record_for_current_school&.training_status_withdrawn?
+
         return true unless transfer?
 
         @email_owner != existing_user
       end
 
+      def email_owner_participant_profile
+        @email_owner.teacher_profile&.ecf_profiles&.first
```

Comment:
`.ecf_profiles` doesn't return withdrawn ones so perhaps `@email_owner.teacher_profile.participant_profiles.ecf` would give you their profiles (they _could_ have more than one though - e.g. ECT and Mentor or active and withdrawn)

Reply:
Ah yeah, that may be better. Here at least I've just replicated exactly what was being used in the now refactored `email_used_in_the_same_school?` method. I didn't want to make any changes at all to any other logic so just broke it up so that this was available to use in `email_owner_participant_profile_withdrawn_from_current_school?`.

Reply:
Think that wasn't (probably incorrectly) considering withdrawn profiles though